core/src/test/resources/server.conf

```diff
@@ -122,6 +122,9 @@ cairo.writer.alter.max.wait.timeout.micro=7770001
 cairo.writer.tick.rows.count=15
 cairo.writer.command.queue.capacity=16
 
```

Comment:
I think these two deserve a comment ! :)

Reply:
> I think these two deserve a comment ! :)

it is really just test config but added a comment, just in case